### PR TITLE
[vcpkg] Unify manifests vcpkg files

### PR DESCRIPTION
**Pull request by talregev**
Unify manifests vcpkg files.

### CODE REVIEW

**Comment by talregev** [comment]
@AenBleidd Ready for review.

File: android/buildAndroidBOINC-CI.sh [hunk 1]
```diff
@@ -119,6 +119,7 @@ if [ "x$build_dir" != "x" ]; then
 else
     cd ../
     BUILD_DIR="$(pwd)/3rdParty/android"
+    TRD_PARTY="$(pwd)/3rdParty"
```
Comment (AenBleidd) [inline android/buildAndroidBOINC-CI.sh:122]: It's not obvious that `TRD_PARTY` means third-party. Please rename to `THIRD_PARTY`

Comment (talregev) [reply android/buildAndroidBOINC-CI.sh:122]: ok.

Comment (talregev) [reply android/buildAndroidBOINC-CI.sh]: done.

**Review by AenBleidd — APPROVED** [review]

**Pull request merged**